cernopendata_client/cli.py

```diff
@@ -84,7 +84,14 @@ def version():
     type=click.STRING,
     help="Which CERN Open Data server to query? [default={}]".format(SERVER_HTTP_URI),
 )
-def get_metadata(server, recid, doi, title, output_value):
+@click.option(
+    "--filter",
+    "filters",
+    multiple=False,
+    help="Filter workflow that contains certain filtering criteria. "
```

Comment:
Moreover, you can enrich the user guide `usage.rst` the section "Getting metadata" so that the David Rousseau example will also appear here: https://cernopendata-client.readthedocs.io/en/latest/usage.html#getting-metadata